docs/Deploying/Debugging.md

```diff
@@ -0,0 +1,158 @@
+# Debugging
+
+## Goals of this guide
+
+This is NOT meant to be an exhaustive list of possible issues in Fleet and how to solve them.
+
+This is a guide for going from a vague statement such as "things are not working correctly" to a more narrowed down and 
+specific assessment. This doesn't mean necessarily a solution; but, with a more specific assessment, it'll be easier for 
+the Engineering team to help.
+
+Note that even if you do all your homework, the Engineering team might have follow-up questions.
+
+## Basic data that is needed
+
+While it's not needed strictly 100% of the times, in most cases it's extremely useful to have a clear understanding of 
+the basic characteristics of the Fleet deployment with the issues:
+
+- Amount of total hosts.
+- Amount of online hosts.
```

Comment:
How about approximate amount of scheduled queries and policies.

Reply:
Also, I wonder if we can make`fleetctl debug archive` answer all these questions for us... (have it generate a report.txt or something with number of users, etc.)